data/commands/rules.json

```diff
@@ -5,5 +5,6 @@
     "{2566BA}- {A9C4E4}The Ship is a peace zone; don't harm players on it and don't run to it if you're fighting!",
     "{2566BA}- {A9C4E4}Do not abuse bugs or commands.",
     "{2566BA}- {A9C4E4}Do not harm a cruise in any way!",
-    "If you have questions, use {D67513}@<message> {A9C4E4}to contact the Las Venturas Playground staff. Have fun!"
+    "{2566BA}- {A9C4E4}Advertising of Server/Website,Is Not Allowed !",
```

Comment:
I'm more than happy to help you make the change. Do you not know how to update the PR? What client are you using?